lib/datadog/notifications/plugins/grape.rb

```diff
@@ -11,11 +13,11 @@ def self.exception_status(err)
     # *<tt>:metric_name</tt> - the metric name, defaults to "grape.request"
     # *<tt>:exception_handler</tt> - a custom exception handler proc which accepts an exception object and returns a status
     # *<tt>:tags</tt> - additional tags
-    def initialize(metric_name: 'grape.request', exception_handler:, **opts)
+    def initialize(metric_name: 'grape.request', exception_handler: DEFAULT_EXCEPTION_HANDLER, **opts)
```

Comment:
Maybe, inlining the proc would be fine as well (const name is almost as long as proc itself)